problems/4.arrays.js

```diff
@@ -2,7 +2,12 @@
 * @param {number[]} arr - The input array
 * @returns {number} - The average of the first and last values
 */
-function firstAndLastAverage() {
+const firstAndLastAverage = (arr) => {
+  if (arr.length < 2) {
+    return null;
+  } else {
```

Comment:
Could eliminate the else b/c we can return from the if, and then just run the next line of code.